docker/development/docker-compose.yml

```
@@ -0,0 +1,13 @@
+version: "3.8"
+
+include:
+  - supabase/docker/docker-compose.yml
+
+services:
+  supabase:
```

I think this should be `superflows`
```suggestion
  superflows:
```